docs/build/messages/reaction.mdx

```diff
@@ -98,7 +98,12 @@ var registry = CodecRegistry()..registerCodec(ReactionCodec());
 
 <TabItem value="rn" label="React Native"  attributes={{className: "rn_tab"}}>
 
-React Native SDK already has the codecs registered.
+```jsx
+const client = await Client.create(wallet, {
```

Comment:
I know I wrote this code but I guess it would probably be more clear if it was named signer.

```suggestion
const client = await Client.create(signer, {
```